cheatsheets/Authentication_Cheat_Sheet.md

```diff
@@ -209,7 +209,13 @@ Multi-factor authentication (MFA) is by far the best defence against the majorit
 
 The [Multifactor Authentication Cheat Sheet](Multifactor_Authentication_Cheat_Sheet.md) contains further guidance on implementing MFA.
 
-#### Account Lockout
+#### Login Throttling
+
+Login Throttling is a protocol used to prevent an attacker from making too many attempts at guessing a password through normal interactive means, it includes:
+
+- Maximum number of attempts.
```

Comment:
This is a good start. Should we perhaps expand on this?

Reply:
Account Lockout is the list item in Login throttling which contains the description

Reply:
I guess for now Its enough to be determined.